README.md

```diff
@@ -1,42 +1,82 @@
-# frappe-docker
-Dockerizing frappe for production
+# Frappe-Docker
 
-Prerequisites
-- ## Install <a href="http://recordit.co/" target="_blank">**Docker**</a>. 
+Dockerizing frappe for production.
 
-## Clone 
-```git@github.com:shridarpatil/frappe-docker.git```
+### Prerequisites:
 
-## Pull docker image
-docker pull shridh0r/frappe:\<tagname\>
+- Install [Docker](https://docs.docker.com/install/)
+- Install [Docker Compose](https://docs.docker.com/compose/install/)
 
-## Run
-```docker-compose up```
+Once docker and docker compose are installed, please follow the below steps.
 
-## Build
-Docker compose build by default pull's frappe master branch </br>
-``` docker-compose build ```
+1. Build the image.
 
-#### Build args
-  - [x] FRAPPE_PATH - Frappe repo path  
-  - [x] FRAPPE_BRANCH - Branch name
-  - [x] FRAPPE_PYTHON - Python version
-  - [x] FRAPPE - Folde name
-  - [x] BENCH_BRANCH - Bench repo path
-  - [x] BENCH_PATH - Branch name
+   Using docker command: `docker-compose build --build-arg FRAPPE_PATH=https://github.com/zerodhatech/frappe.git --build-arg FRAPPE_BRANCH=<branch_name> .`
 
-Set frappe-path/branch dynamically by passing build-arg
-```
-docker-compose build --build-arg FRAPPE_PATH=https://github.com/zerodhatech/frappe.git --build-arg FRAPPE_BRANCH=zero_v12
-```
+   Using docker-compose: `docker-compose build`
 
-## Run
-```docker-compose up```
+   This is assuming you are running build from same directory as Dockerfile.
 
-## Create site
-Exec into docker container and create new site
+   More on [build phase](#build-phase).
 
-Create new-site
-```
-bench new-site site1.local --force --db-type postgres --db-root-username postgres --db-root-password root
+2. Run the image.
+
+   using docker-compose: `docker-compose up`
+
+   More on [run phase](#run-phase).
+
+The image is also present at [Dockerhub](https://hub.docker.com/r/shridh0r/python-ubuntu) which can be pulled and executed.
+
+### Creating site
+
+The frappe container created from above will not have sites folder yet. Inorder to create a site, execute the startup script as `/bin/sh start_up.sh new_site <arguments>`.
```

Comment:
Sites folder will be there but there will be no site.

1. Site creation should be from bench new-site command
1. We should explain to exec and then create new-site